x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -808,7 +808,13 @@ public enum Cap {
          * and https://github.com/elastic/elasticsearch/issues/120803
          * Support for queries that have multiple SORTs that cannot become TopN
          */
-        REMOVE_REDUNDANT_SORT;
+        REMOVE_REDUNDANT_SORT,
+
+        /**
+         * Fixes a series of issues with inlinestats which had an incomplete implementation after lookup and inlinestats
+         * were refactored.
+         */
+        INLINESTATS_STUBRELATION_PROPER_REPLACEMENT;
```

Comment:
Maybe pick a smaller name that we can version while iterating on the implementation, for example: INLINESTATS_V2 ?